idaes/models_extra/power_generation/costing/BB_costing_parameters.json

```diff
@@ -1 +1 @@
": 0.0, "Project Contingency": 0.15, "RP Value": 277700.0, "Units": "kW"}, "8.2": {"BEC": 5967.848780853654, "Eng Fee": 0.15, "Process Contingency": 0.0, "Project Contingency": 0.15, "RP Value": 277700.0, "Units": "kW"}, "8.3": {"BEC": 10971.457829529923, "Eng Fee": 0.15, "Process Contingency": 0.0, "Project Contingency": 0.15, "RP Value": 1414.24819, "Units": "MBtu/hr"}, "8.4": {"BEC": 10583.246467542695, "Eng Fee": 0.15, "Process Contingency": 0.0, "Project Contingency": 0.25, "RP Value": 1576062.15, "Units": "lb/hr"}, "8.5": {"BEC": 784.343631531533, "Eng Fee": 0.15, "Process Contingency": 0.0, "Project Contingency": 0.3, "RP Value": 277700.0, "Units": "kW"}, "9.1": {"BEC": 15496.994810424325, "Eng Fee": 0.15, "Process Contingency": 0.0, "Project Contingency": 0.15, "RP Value": 2298.612246483964, "Units": "MBtu/hr"}, "9.2": {"BEC": 1662.336586369446, "Eng Fee": 0.15, "Process Contingency": 0.0, "Project Contingency": 0.15, "RP Value": 229862.1440969728, "Units": "gal/min"}, "9.3": {"BEC": 12131.493920788585, "Eng Fee": 0.15, "Process Contingency": 0.0, "Project Contingency": 0.15, "RP Value": 229862.1440969728, "Units": "gal/min"}, "9.4": {"BEC": 11385.487870057312, "Eng Fee": 0.15, "Process Contingency": 0.0, "Project Contingency": 0.2, "RP Value": 229862.1440969728, "Units": "gal/min"}, "9.5": {"BEC": 1497.0008331824197, "Eng Fee": 0.15, "Process Contingency": 0.0, "Project Contingency": 0.2, "RP Value": 5196.768864894039, "Units": "gal/min"}, "9.6": {"BEC": 653.7025792881713, "Eng Fee": 0.15, "Process Contingency": 0.0, "Project Contingency": 0.2, "RP Value": 229862.1440969728, "Units": "gal/min"}, "9.7": {"BEC": 1369.338982446314, "Eng Fee": 0.15, "Process Contingency": 0.0, "Project Contingency": 0.3, "RP Value": 229862.1440969728, "Units": "gal/min"}, "10.1": {"BEC": 2975.2252770357054, "Eng Fee": 0.15, "Process Contingency": 0.0, "Project Contingency": 0.15, "RP Value": 590.5718256, "Units": "ton/d"}, "10.2": {"BEC": 1685.3960415924944, "Eng Fee": 0.15, "Process Contingency": 0.0, "Project Contingency": 0.15, "RP Value": 590.5718256, "Units": "ton/d"}, "10.3": {"BEC": 757.5010160187344, "Eng Fee": 0.15, "Process Contingency": 0.0, "Project Contingency": 0.15, "RP Value": 590.5718256, "Units": "ton/d"}, "10.6": {"BEC": 2359.4785625848244, "Eng Fee": 0.15, "Process Contingency": 0.0, "Project Contingency": 0.15, "RP Value": 590.5718256, "Units": "ton/d"}, "10.7": {"BEC": 538.7718848531108, "Eng Fee": 0.15, "Process Contingency": 0.0, "Project Contingency": 0.15, "RP Value": 590.5718256, "Units": "ton/d"}, "10.8": {"BEC": 162.2277636863303, "Eng Fee": 0.15, "Process Contingency": 0.0, "Project Contingency": 0.15, "RP Value": 590.5718256, "Units": "ton/d"}, "10.9": {"BEC": 1032.2718711308605, "Eng Fee": 0.15, "Process Contingency": 0.0, "Project Contingency": 0.3, "RP Value": 590.5718256, "Units": "ton/d"}, "11.1": {"BEC": 4773.257359744647, "Eng Fee": 0.15, "Process Contingency": 0.0, "Project Contingency": 0.15, "RP Value": 277700.0, "Uni
```

Comment:
can you please update the dictionary adding a new key - CE_index_base_year ?
All these accounts are dollars 2018
Maybe this info belongs to BB_costing_exponents.json

Reply:
I'll update BB_costing_parameters.json, the custom_costing_params dictionary and sCO2_costing_params for the appropriate base years. I don't think the exponents need it, they are not dependent on the currency units.